webapp-django/crashstats/crashstats/decorators.py

```diff
@@ -65,3 +68,21 @@ def inner(request, *args, **kwargs):
             raise
         return view(request, *args, **kwargs)
     return inner
+
+
+def track_api_pageview_decorator(view):
```

Comment:
Isn't it weird to put ``decorator`` in the name of a decorator, in a ``decorators.py`` file?